### PR TITLE
Document Preview windows closing (not displaying).

### PR DESCRIPTION
**Pull request by marxin**
Explains better #650.

### CODE REVIEW

File: doc/vim-lsp.txt [hunk 1]
```diff
@@ -224,6 +224,11 @@ g:lsp_preview_keep_focus                         *g:lsp_preview_keep_focus*
 	let g:lsp_preview_keep_focus = 0
 <
     * |preview-window| can be closed using the default vim mapping - `<c-w><c-z>`.
+    * preview-window can be also automatically closed after completion with
```
Comment (prabirshrestha) [inline doc/vim-lsp.txt]: need to include `|preview-window|` with pipe so can navigate easily to more details about preview window.

Comment (marxin) [reply doc/vim-lsp.txt]: Fixed. Btw. can you please send me a link to the VIM documentation markup language syntax?

**Comment by marxin** [comment]
May I please ping this.

**Comment by marxin** [comment]
PING^2

**Comment by hrsh7th** [comment]
I think it can be merged.
Thank you.

**Pull request merged**